scripts/Calc_hru_wgtavg_nc.py

```diff
@@ -234,8 +172,21 @@ def usage():
       varname = args[2]  
       nc_out  = args[3]
 
-      wgtVal=compAvgVal(nc_wgt,nc_in,varname)
-      writeNetCDFData(nc_out, wgtVal, varname)     
+      func=partial(compAvgVal, nc_wgt, nc_in, varname)
+
+      pool = mp.Pool(processes=NPROC,maxtasksperchild=1)
+      results=pool.map(func,range(NPROC))
+      result=np.column_stack(results)
+
+      with xr.open_dataset(nc_in) as ds:
+        timedata  = ds.coords[TDIM]     # Get time coordinate variable
```

Comment:
you can probably drop the `coords` part here. You'll get the time variable either way but dropping `coords` will protect against a malformed input file that isn't decoded having the time variable as a coordinate.

Reply:
also, `TDIM`can probably be renamed to `TIME_VAR` or something more descriptive/accurate.